alvr/audio/src/linux.rs

```diff
@@ -218,7 +218,9 @@ pub fn record_audio_blocking_pipewire(
     let is_running_clone_for_pw_terminate: Arc<dyn Fn() -> bool + Send + Sync> =
         Arc::clone(&is_running);
     thread::spawn(move || {
-        while is_running_clone_for_pw_terminate() {}
+        while is_running_clone_for_pw_terminate() {
+            thread::sleep(KEEPALIVE_INTERVAL);
```

Comment:
I'm a bit conflicted with the naming for the duration constant. We use `KEEPALIVE_INTERVAL` only in two instances, in the keepalive loop in the server and client. this is not a keepalive thread. Well the function of the loop is, but with slight different meaning.

Also the name for the function pointer is way too verbose, scoping + shadowing should have been used, but that's not in the scope of this PR.

Reply:
I think we should make a new Duration constant local to the file, with a more precise name, like `TERMINATION_WAIT_INTERVAL`

Reply:
We don't really have any other ones floating around and I didn't really want to make a new one.

Reply:
For correctness we should make a new one. They have different scopes.